templates/store/store.html

```diff
@@ -31,59 +31,55 @@ <h2>Big data</h2>
   </div>
 </div>
 
-<div class="p-strip is-shallow u-no-padding--bottom">
+<div class="p-strip--accent is-slanted">
   <div class="row">
-    <div class="u-equal-height">
-      <div class="col-6 p-card">
+    <div class="col-6">
       <div class="col-2 u-align--center">
-        <img src="https://assets.ubuntu.com/v1/96c61b0d-charm-icon.png" alt="Illustration of a Charm pictogram">
+        <img src="https://assets.ubuntu.com/v1/96c61b0d-charm-icon.png" alt="">
```

Comment:
Why remove the alt?

Reply:
Because its a decorative image that adds no value to the content it accompanies.